README.rst

```diff
@@ -1,3 +1,98 @@
 =============
 python-bleson
-=============
\ No newline at end of file
+=============
+
+Status
+======
+
+Experimental.
+
+
+Public API
+==========
+
+Outline of the 'bleson' Python packages an end-user would use to play with Bluetooth LE.
+
+.. code:: python
+
+    bleson
+        get_provider()          # Obtain the provider for the current platfrom (currently Linux HCI only)
+
+    bleson.core.types           # Value objects representing Bluetooth Core Spec data types (not HCI specific)
+        BDAddress
+        Device
+        Advertisement
+
+    bleson.core.roles           # Core Bluetooth roles
+        Advertiser
+        Observer
+
+    bleson.beacons.eddystone
+        EddystoneBeacon         # Physical Web beacon, is a subclass of Advertiser role
+
+    bleson.logger
+        log                     # Simple convenience wrapper around Python's standard logging.
+
+
+
+Examples
+========
+
+Examples are abbreviated below, plese see 'examples' folder for more details.
+
+Example - Advertiser
+--------------------
+
+.. code:: python
+
+    from bleson import get_provider, Advertiser, Advertisement
+
+    adapter = get_provider().get_adapter()
+    advertiser = Advertiser(adapter)
+    advertisement = Advertisement(name = "bleson")
+    advertiser.start()
+    advertiser.stop()
+
+
+Example - Eddystone Beacon
+--------------------------
+
+.. code:: python
+
+    from bleson import get_provider, EddystoneBeacon
+
+    adapter = get_provider().get_adapter()
+    beacon = EddystoneBeacon(adapter)
+    beacon.url = 'https://www.bluetooth.com/'
+    beacons.start()
+    beacons.stop()
+
+
+Example - Observer
```

Comment:
Not many adverts were turning up.
I tried getting my phone to advertise and that didn't turn up at all.
When I did beacons it would turn up twice and then no more showed up.